### PR TITLE
PICARD-760: Update logo and derived icons

### PR DESCRIPTION
**Pull request by Freso**
Details in individual commits.

### CODE REVIEW

**Comment by Sophist-UK** [comment]
I was wondering why the new logos do not have a music symbol overlayed as this is a significant symbol for Picard.

Any chance that we can get a music symbol overlayed onto the new icons?

File: AUTHORS.txt [hunk 1]
```diff
@@ -2,7 +2,6 @@ Lukáš Lalinský <lalinsky@gmail.com>
 Robert Kaye <rob@eorbit.net>
 Philipp Wolfer <ph.wolfer@googlemail.com>
 
-Picard icons by Carlin Mangar <openshift@gmail.com>
-
-The icons in this program are from the Tango Desktop Project
-<http://tango.freedesktop.org/>.
+Picard logo by <http://monkeydo.digital>, © 2015 MetaBrainz Foundation
+Some icons in this program are by Carlin Mangar <openshift@gmail.com>, the
```
Comment (mineo) [inline AUTHORS.txt]: "Some icons" seems very vague.

Comment (Freso) [inline AUTHORS.txt]: It is very vague, but IMO not more vague than what was already there. I looked over commits he has done, but he doesn't state on the icons he's added whether he's actually the creator of them or if they're taken from the Tango set (or if they're from some third place).

**Comment by Freso** [comment]
> Any chance that we can get a music symbol overlayed onto the new icons?

This should probably have been brought up when the new icons were being thrown around. The icons I've added here are the official Picard icons. A music overlay would interfere with the "brand" of the icon.

File: AUTHORS.txt (hunk 1 above)
Comment (phw) [inline AUTHORS.txt]: From memory: He created the logo and the empty CD case shown if no cover art is availableand the AcoustId (back then it were still PUIDs) submission icon as it is now (this was an updated version of the existing icon, see https://github.com/musicbrainz/picard/commit/3faab92ef96707ff1e68c50b61df403864c49325)

@Freso Was there anything else you saw?

Comment (Freso) [inline AUTHORS.txt]: @phw https://ptpb.pw/HlCS

Comment (Freso) [inline AUTHORS.txt]: Anyway, AUTHORS.txt doesn't seem, to me, to be meant as a detailed declaration of authorship/copyright, but just meant to give an overview. If you want to know specifics, `git log` or `git blame` would be what you're after.

Comment (Freso) [inline AUTHORS.txt]: How about something like

> Interface icons by CM and from the Tango Project.

?

**Comment by phw** [comment]
Looks good, I actually like how it looks with the new logo on my system. Some notes:

* You should run `python setup.py build_ui` to update `picard/resources.py`
* A couple if images in [`installer/images`](https://github.com/musicbrainz/picard/tree/master/installer/images) also use the logo (for the Windows installer). A SVG file is provided so this should be easy to update

File: AUTHORS.txt (hunk 1 above)
Comment (phw) [inline AUTHORS.txt]: Yeah, it's still vague but sounds better ;) I think that's fine.

**Comment by phw** [comment]
> I was wondering why the new logos do not have a music symbol overlayed as this is a significant symbol for Picard.

I don't think the logo will be changed again at this point, this was the final design. The logo together with a note is already the official icon for MusicBrainz itself:

MusicBrainz Logo

Also just adding the note as an overlay would not really work well with the brand design represented by the whole set of MetaBrainz logos.

**Comment by Sophist-UK** [comment]
I have missed the whole set of logo discussions when they happened. But the MB logo does have a music symbol, so my suggestion would be to keep the music symbol and put the tag symbol to replace the circuit-board half of the logo rather than the music symbol half.

You would then have something like this which effectively says "music tagger"...

picard-logo-mockup

**Comment by phw** [comment]
If you look over the logos for all the projects (see the logo carousel at https://metabrainz.org/) you will notice that the right side is the common element for all those logos. I have no say on this anyway, but I don't think this is the right place to discuss this. If you think it is important to reconsider this the mailing list would be a better place for discussion.

**Comment by Sophist-UK** [comment]
I had a feeling that this might be the answer. Its a minor point anyway.

**Comment by Freso** [comment]
> A couple if images in installer/images also use the logo

Not sure if medium or high detail version should be used for this: https://imgur.com/a/v6npE

1. Medium detail
2. High detail

Feedback/thoughts?

**Comment by Freso** [comment]
> You should run `python setup.py build_ui` to update `picard/resources.py`

```
freso@koume> python2 setup.py build_ui
running build_ui
/usr/bin/pyrcc4 /home/freso/Development/MusicBrainz/picard/resources/picard.qrc -o /home/freso/Development/MusicBrainz/picard/picard/resources.py
pyrcc4 Parse Error:/home/freso/Development/MusicBrainz/picard/resources/picard.qrc:1:1 [unexpected end of file]
command '/usr/bin/pyrcc4' failed with exit status 1
```

Any hints? (`resources/picard.qrc` seems to be empty.)

**Comment by Freso** [comment]
I think all resolvable comments have been addressed now. :)

**Comment by smaragdus** [comment]
The old icon (as of version 1.3,2) was cute and nice, the new one is disgustingly flat and unbearably ugly to me so that I reverted to the previous version- I cannot tolerate flat ugliness no matter how much better the new version (1.4) might be.

**Pull request merged**